src/search-core-re-exports.ts

```diff
@@ -16,6 +16,7 @@ export {
   Context,
   ClientSDKHeaderValues,
   CloudRegion,
+  CloudChoice,
```

Comment:
Since this is mostly alphabetical (it's not entirely though), should choice be before region then?

Reply:
Done